addons/web/controllers/main.py

```diff
@@ -2039,7 +2037,10 @@ def report_download(self, data, token, context=None):
 
         :param data: a javascript array JSON.stringified containg report internal url ([0]) and
         type [1]
-        :returns: Response with a filetoken cookie and an attachment header
+        :returns: Response with an attachment header
+
+        Token is deprecated and no more used
```

Comment:
Note is probably unnecessary, or if present should probably be the documentation for `:param token:` (which is currently undocumented)